types/src/signature.rs

```diff
@@ -0,0 +1,69 @@
+//! Platform wrappers for tendermint's [`Signature`]
+//!
+//! [`Signature`]: tendermint::signature::Signature
+
+#[cfg(all(target_arch = "wasm32", feature = "wasm-bindgen"))]
+pub use wbg::*;
+
+#[cfg(all(target_arch = "wasm32", feature = "wasm-bindgen"))]
+mod wbg {
+    use js_sys::Uint8Array;
+    use tendermint::signature::Signature;
+    use wasm_bindgen::prelude::*;
+
+    /// Signature
+    #[derive(Clone, Debug)]
+    #[wasm_bindgen(getter_with_clone, js_name = "Signature")]
+    pub struct JsSignature(pub Uint8Array);
```

Comment:
is `pub` needed for `Uint8Array`?

Reply:
This is mostly meant as a type returned to JS, so I've left it public so that JS side can inspect the contents.